packages/services/src/services/Locale/Locale.js

```diff
@@ -114,8 +114,8 @@ class LocaleAPI {
       const lang = root.document.documentElement.lang.toLowerCase();
       const codes = lang.split('-');
       return { cc: codes[1], lc: codes[0] };
-      //    } else {
-      //      return _localeDefault;
```

Comment:
oh can you also add check here that both `cc` and `lc` are defined from the `lang` attr? If either are undefined then we'll return default for now. For cases where the `lang` is just "en" like we saw from one of the adopters.

Reply:
Yup, added here: https://github.com/carbon-design-system/ibm-dotcom-library/pull/696/commits/e5d31ec15b476461aa0a5d9ee3899b85d83db647